sphinxserve/__init__.py

```diff
@@ -172,13 +173,17 @@ def manage(self):
         '''Manage web server, watcher and sphinx docs renderer
         '''
         result = self.build()
-
         if result.code != 0:
             sys.exit(result.stderr)
-
         logger.debug(result.stdout)
         workers = [spawn(self.serve), spawn(self.watch), spawn(self.render)]
+
+        def shutdown_handler():
```

Comment:
This code looks great.
Just a minor detail, moving this shutdown_handler function right after
the manage method docstring would read better.